faq.md

```diff
@@ -482,16 +482,14 @@ what you might need reflection/mirrors for – please let us know at
 
 ### How do I do internationalization (i18n), localization (l10n), and accessibility (a11y) in Flutter?
 
-Flutter has basic support for accessibility on iOS and Android, though this
-feature is a work in progress.
+Learn more about i18n and l10n in the
+[Flutter Internationalization Tutorial](https://flutter.io/tutorials/internationalization/).
```

Comment:
Please remove https://flutter.io from the URL.

As it is, it makes it hard to stage the in-progress site.